website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -9,6 +9,65 @@
 
 - Use helper text to give the user extra details about the data you’re asking them to input, e.g., formatting requirements such as MM-DD-YYYY.
 
+### Form::CharacterCount
+
+!!! Info
+
+The character count is not coupled with the invalid state of the field. Instead, it is the responsibility of the consumer to implement validation at the application-level.
+!!!
+
+Use a character count to communicate the current length of the value in an input and whether it meets or exceeds the length requirements passed to the component. The component accepts multiple arguments to set length requirements and exposes several computed values to support custom messages. Visit the [code](/components/form/primitives?tab=code#formcharactercount-1) documentation for more details.
+
+#### Default messages
+
+Depending on which property (or properties) are passed to the component, a different default message will be displayed by the component to communicate the relationship between the current length of the input value (`currentLength`) and the maximum length (`maxLength`), minimum length (`minLength`), or both.
+
+<video width="100%" controls loop>
+  <source
+    src="/assets/components/form/primitives/character-count-default-interactions.mp4"
+    type="video/mp4"
+  />
+</video>
+
+!!! Info
+
+Test and interact with the default messaging examples in the [code](/components/form/primitives?tab=code#formcharactercount-1)
+!!!
```

Comment:
There are quite a few info banners in this section and it makes it hard to know what info is really important. Is this one really necessary? Alternatively, you could drop the actual component into the page so they can play with it directly here.

Reply:
Removed this as a banner in https://github.com/hashicorp/design-system/pull/1799/commits/667178541151a2700d6e41709f0bffd97d8d0404